### PR TITLE
Add /whoami

### PR DESCRIPTION
**Pull request by Protonull**

### CODE REVIEW

**Comment by Protonull** [comment]
Is useful for people with [account switch mods](https://www.curseforge.com/minecraft/mc-mods/in-game-account-switcher), bots, or are using Kira.

File: src/main/java/vg/civcraft/mc/civchat2/commands/WhoAmI.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import vg.civcraft.mc.civmodcore.command.StandaloneCommand;
+import vg.civcraft.mc.namelayer.NameAPI;
+
+@CivCommand(id = "whoami")
+public class WhoAmI extends StandaloneCommand {
+
+	@Override
+	public boolean execute(CommandSender sender, String[] arguments) {
+		final String response = ChatColor.YELLOW + "You are: " + ChatColor.RESET;
+		if (sender instanceof Player) {
```
Comment (Maxopoly) [inline src/main/java/vg/civcraft/mc/civchat2/commands/WhoAmI.java:19]: Kinda weird to explicitly allow this command for server console in the plugin.yml, but then make it not work for it, it'd just say `You are: `. I'd just disallow it for non-players?

Comment (Protonull) [reply src/main/java/vg/civcraft/mc/civchat2/commands/WhoAmI.java:19]: What do you mean? Does the console not have a name?

Comment (Maxopoly) [reply src/main/java/vg/civcraft/mc/civchat2/commands/WhoAmI.java:19]: Ah nevermind I cant read

**Comment by Maxopoly** [comment]
Nice work

**Pull request merged**